.appveyor.yml

```diff
@@ -1,4 +1,4 @@
-﻿version: 2.3.2.{build}
+version: 2.3.2.{build}
```

Comment:
this is probably just a line-ending change. I'll try to revert it

Reply:
Found it! It was this https://www.freecodecamp.org/news/a-quick-tale-about-feff-the-invisible-character-cd25cd4630e7/

![image](https://user-images.githubusercontent.com/7608904/79292176-18884780-7e96-11ea-8be9-2dce6526beed.png)